scripts/setup-versions.sh

```diff
@@ -37,7 +34,7 @@ GFLAGS_VERSION="v2.2.2"
 GLOG_VERSION="v0.6.0"
 LZO_VERSION="2.10"
 SNAPPY_VERSION="1.1.8"
-THRIFT_VERSION="v0.16.0"
+THRIFT_VERSION="v0.21.0"
```

Comment:
This does not match the arrow version or was this related to the fbthrift stuff?

Reply:
We should stick to thrift  v0.16.0 which aligns with the Arrow version we use when bundling.
manylinux is likely out of sync.

Reply:
Updated the version in setup-versions.sh and passed it separately in manylinux.